jvb/src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java

```diff
@@ -151,28 +159,15 @@ public void run()
          if (paddingBps < 1)
          {
              // Not much.
+             bytesLeftOver = 0;
              return;
          }
 
+         long bytes = (config.getPaddingPeriodMs() * paddingBps / 1000 / 8) + bytesLeftOver;
```

Comment:
Merely in the interest of not changing too many things at once, I'm wondering if we should hold off on the `bytesLeftOver` as part of this fix.

Reply:
Ok, I can separate that.